src/commands/tokens/mod.rs

```diff
@@ -83,3 +83,50 @@ impl TokensActions {
         }
     }
 }
+
+async fn params_ft_metadata(
+    config: crate::config::Config,
+    ft_contract_account_id: crate::types::account_id::AccountId,
+    network_config: crate::network_view_at_block::NetworkViewAtBlockArgs,
+) -> color_eyre::eyre::Result<(u64, String)> {
+    let query_view_ft_metadata_response = network_config
+        .get_network_config(config.clone())
+        .json_rpc_client()
+        .call(near_jsonrpc_client::methods::query::RpcQueryRequest {
+            block_reference: network_config.get_block_ref(),
+            request: near_primitives::views::QueryRequest::CallFunction {
+                account_id: ft_contract_account_id.clone().into(),
+                method_name: "ft_metadata".to_string(),
+                args: near_primitives::types::FunctionArgs::from(vec![]),
+            },
+        })
+        .await
+        .map_err(|err| {
+            color_eyre::Report::msg(format!("Failed to fetch query for view method: {:?}", err))
+        })?;
+    let call_result =
+        if let near_jsonrpc_primitives::types::query::QueryResponseKind::CallResult(result) =
+            query_view_ft_metadata_response.kind
+        {
+            result.result
+        } else {
+            return Err(color_eyre::Report::msg("Error call result".to_string()));
+        };
+    let ft_metadata = if call_result.is_empty() {
+        serde_json::Value::Null
+    } else {
+        serde_json::from_slice(&call_result)
+            .map_err(|err| color_eyre::Report::msg(format!("serde json: {:?}", err)))?
+    };
+    let decimals = ft_metadata
+        .get("decimals")
+        .expect("Impossible to get decimals!")
+        .as_u64()
+        .expect("Impossible represent it as u64!");
+    let symbol = ft_metadata
+        .get("symbol")
+        .expect("Impossible to get symbol!")
+        .as_str()
+        .expect("Impossible represent it as &str!");
```

Comment:
I would use a struct to deserialize it in one go and handle errors better:

```rust
#[derive(serde::Deserialize)]
struct FtMetadata {
    symbol: String,
    decimals: u64,
}
```

Reply:
done